tools/lib/src/device.dart

```diff
@@ -272,6 +272,45 @@ class EmulatorDevice extends Device {
     );
   }
 
+  /// Grants all privacy-related permissions to apps by default.
+  ///
+  /// This only applies to apps newly installed to the device after this call.
+  bool _disablePermissionPopups() {
+    if (!isConnected) {
+      return false;
+    }
+    final Map<String, String> capability = _tizenSdk.sdbCapability(serial!);
+    final bool rooted = capability['rootonoff_support'] == 'enabled';
+    if (!rooted) {
+      return false;
+    }
+
+    io.ProcessResult result = _processRunner.runSync(
+      _tizenSdk.sdb.path,
+      <String>['-s', serial!, 'root', 'on'],
+    );
+    if (result.exitCode != 0) {
+      print('Error: running "sdb root on" failed.');
+      return false;
+    }
+
+    result = _processRunner.runSync(
+      _tizenSdk.sdb.path,
+      <String>['-s', serial!, 'shell', 'touch', '/opt/share/askuser_disable'],
```

Comment:
This cannot be done in driver scripts because the file (`askuser_disable`) must be created even before the app is installed.